e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -10,6 +10,43 @@ export function getDashboardCard(index = 0) {
   return cy.get(".DashCard").eq(index);
 }
 
+function getDashCardApiUrl(dashId) {
+  return `/api/dashboard/${dashId}/cards`;
+}
+
+const DEFAULT_CARD = {
+  id: -1,
+  row: 0,
+  col: 0,
+  size_x: 8,
+  size_y: 8,
+  visualization_settings: {},
+  parameter_mappings: [],
+};
+
+export function addCardToDashboard({ card_id, dashboard_id, card }) {
+  return cy
+    .request("PUT", getDashCardApiUrl(dashboard_id), {
+      cards: [
+        {
+          ...DEFAULT_CARD,
+          card_id,
+          ...card,
+        },
+      ],
+    })
+    .then(response => ({
+      ...response,
+      body: response.body[0],
+    }));
+}
+
+export function addCardsToDashboard({ dashboard_id, cards }) {
```

Comment:
nit: wdyt about naming it like `updateDashboardCards` and adding a jsdoc describing it is suitable for cards CRUD since `addCardsToDashboard`  makes it not so obvious it can be used for updating/removing cards

Reply:
Yup that sounds better to me, I can rename it

Reply:
It's been renamed on the latest commit